app/views/orgs/rosters/select_google_classroom.html.erb

```diff
@@ -0,0 +1,38 @@
+<% view = FormView.new(subject: @roster) %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h1 class="site-content-heading">Add students</h1>
+  </div>
+
+  <div class="site-content-body">
+    <div class="markdown-body">
+      <p>Let's set up a roster so you can easily track student progress on your dashboard. You can always come back and complete this step later.</p>
+    </div>
+
+    <div class="my-3">
+      <h2 class="h3 mb-2">Select the Google Classroom roster you want to import</h2>
+
+      <div class="Box overflow-auto">
+        <ul>
+          <% @google_classroom_courses.each do |course| %>
+            <li class="Box-row">
+              <label>
+                <input type="checkbox" class="mr-2" checked="checked" aria-describedby="help-text-for-checkbox">
+                <%= course.name %>
+                <%# <%= link_to course.name, {controller: "rosters", action: :import_from_google_classroom, course_id: course.id}, method: :patch %>
+              </label>
+            </li>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+
+    <div class="f4 text-gray text-italic my-3">or <a href="" class="btn-link">manually add students</a></div>
```

Comment:
This needs to be linked back to the default "manual" view for creating a roster.